.circleci/config.yml

```diff
@@ -123,14 +124,14 @@ jobs:
     executor: macos
     steps:
       - checkout
+      - install_macos_dependencies
```

Comment:
Moved this back to its original place, since I needed to install python `requests` package and thought it'd be better to keep all installs at 1 place. This does add ~2+ mins before the skip CI script can run. But I guess that shouldn't be a big deal.